code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -224,12 +248,18 @@
 		new /obj/item/stack/cable_coil(get_turf(src), 10)
 		slung = 0
 		update_icon()
+		worn_icon_state = "gun"
+		worn_icon = null
```

Comment:
this should already be handled by the base sawoff proc.

Reply:
Reverted. Updated the base sawoff proc to have the worn_icon_state set to "gun".